contrib/masterfiles-stage/common.sh

```diff
@@ -59,6 +59,7 @@ git_setup_local_mirrored_repo() {
   # /opt/cfengine/git_mygitserver.net_joeblow_my_policy_repo.git
 
   if [ -d "${local_mirrored_repo}" ] ; then
+    git --git-dir="${local_mirrored_repo}" fetch --dry-run # to provide information in logs about from and to commits
```

Comment:
How does this make it logged more than the run without `--dry-run` below?

Reply:
I suppose that's true. I wasn't seeing anything without this.